### PR TITLE
Bump version 2.5.0dev

### PR DESCRIPTION
**Pull request by d4straub**
After release PR to update pipeline version & fix typo in an error message, reported in https://github.com/nf-core/ampliseq/issues/485

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -252,7 +252,7 @@ manifest {
     description     = 'Amplicon sequencing analysis workflow using DADA2 and QIIME2'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.10.3'
-    version         = '2.4.0'
+    version = '2.5.0dev'
```
Comment (erikrikarddaniel) [inline nextflow.config:255]: Should the `=` be inline with the ones above?

**Review by erikrikarddaniel — APPROVED** [review]
Except for a formatting question, great work!

File: nextflow.config (hunk 1 above)
Comment (d4straub) [reply nextflow.config:255]: Absolutely, thanks, will correct it.

Comment (d4straub) [reply nextflow.config]: done

**Comment by d4straub** [comment]
Thanks for the review!

**Pull request merged**